src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/MembersMustExist.cs

```diff
@@ -35,17 +35,11 @@ private void RunOnTypeSymbol(ITypeSymbol left, ITypeSymbol right, IList<CompatDi
         {
             if (left != null && right == null)
             {
-                AddDifference(left, DifferenceType.Removed, Resources.TypeExistsOnLeft);
+                differences.Add(CreateDifference(left, DiagnosticIds.TypeMustExist, DifferenceType.Removed, "Type '{0}' exists on the left but not on the right"));
```

Comment:
Shouldn't these all be in the resx?

Reply:
Looks like we have TypeExistsOnLeft or MemberExistsOnLeft etc.

Reply:
Yes. This was because I had started the work before I added the resx. Good catch.

Reply:
Fixed, thanks for catching this 😄